client/build-config/src/webpack/commonConfig.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'path'
+
+import TerserPlugin from 'terser-webpack-plugin'
+import webpack, { Configuration } from 'webpack'
+
+import { ROOT_PATH } from '../paths'
+
+export const getCacheConfig = (...invalidateCacheFiles: string[]): Configuration['cache'] => ({
```

Comment:
Let's make this argument an object to improve the readability in consumers; otherwise, it's not clear what arguments are for without digging into the implementation.

```suggestion
interface GetCacheConfigOptions {
  invalidateCacheFiles: string[]
}

export const getCacheConfig = ({ invalidateCacheFiles }: GetCacheConfigOptions): Configuration['cache'] => ({
```